### ISSUE TITLE
Types.Files development

### ISSUE THREAD

**Issue by smeeteday** [keystonejs/keystone#4590]
keystonejs is a very nice CMS system. 
The only regret is that implementations of types.file and types.files have not yet been released.
When will the development of types.files be done?

**Comment by laurenskling** [comment]
There is a type for file right?
http://keystonejs.com/docs/database/#fieldtypes-localfile

**Comment by htor** [comment]
yes, there is a file type https://github.com/keystonejs/keystone/tree/master/fields/types/file and no, not a files type yet.

**Comment by wassimdevox** [comment]
+1

**Comment by myadzel** [comment]
No need for a new type. Just need implement "multiple" option to File.

**Comment by myadzel** [comment]
+1
